README.md

```diff
@@ -34,7 +34,7 @@ Bug reports in JIRA for all driver projects (i.e. NODE, PYTHON, CSHARP, JAVA) an
 Core Server (i.e. SERVER) project are **public**.
 
 ## Installation
-The driver supports use with Swift 5.1+ on MacOS and Linux.
+The driver supports use with Swift 5.1+. The minimum macOS version required to build the driver is 10.14. The driver is tested in continuous integration against macOS 10.14, Ubuntu 16.04, and Ubuntu 18.04.
```

Comment:
So this mentions the minimum OS versions, but what is our OS support policy going forward? The last two macOS releases? I think for Linux we don't really need to do anything special since the runtime is bundled with the Swift distributions as we saw earlier.

Reply:
I don't know if we can really state a policy in that way since it depends on our major version release cadence as well as macOS release cadence. for now we support the last two versions... when 10.16 comes out we'll support that too and won't be able to drop 10.14 until 2.0. 
in practice I think this will likely translate to usually being ~ the last 2 releases, but not always. we could say at least the last 2, but imagine the following scenario: 10.16 is the latest release but came out several months months ago, and there is some feature/library we really want that is only available in 10.16+. we are about to tag 2.0 and probably won't tag 3.0 for a couple more years. do we drop 10.15? (I'd say yes)

Reply:
Ah okay, makes sense. And I know I've asked this before (maybe multiple times) so I apologize for forgetting, but why do we need/want to support 10.14 still?

Reply:
we can't actually test on 10.15 right now because evergreen has no hosts for it, so that eliminates the possibility of putting 10.15 as the minimum version in our `Package.swift`, as the driver would no longer compile on the 10.14 hosts.

that said we *could* have this at 10.14 but just say that we don't support 10.15, but I think that's confusing and users could easily miss it.
we don't really have a compelling reason to drop 10.14 right now anyway other than eventually needing `SwiftCrypto`, but my expectation is that we will be releasing any changes using that in 2.0 anyway.